packages/tests-e2e/src/utils/constants.ts

```diff
@@ -53,6 +52,9 @@ export enum AuthType {
 
 export const emailLinkUrlToken = 'UaTwjBJwyDLMGVbR7WHh';
 
+export const microsoftEmail = 'corbado.systemtest@outlook.com';
```

Comment:
I think we should inject those secrets through ENV variables so that the account is not being abused

Reply:
Should we also add them to github secrets for the CI environment?